integ/workflow/locking/routers.go

```diff
@@ -107,6 +111,19 @@ func (h *handler) ApiV1WorkflowWorkerRpc(c *gin.Context) {
 		})
 		return
 	}
+
+	signalChannelInfo := (*req.SignalChannelInfos)[UnusedSignalChannelName]
+	if signalChannelInfo.GetSize() != 4 {
```

Comment:
I am not following this part. What is this check? Why the size is expected to be 4?

Reply:
I just added a few comments to the test. 

The signals are sent in a different file `locking_test.go`. 

Changed to a constant to be more clear Lol.